package.json

```diff
@@ -21,7 +21,7 @@
     "bowser": "1.6.0",
     "crypto-md5": "^1.0.0",
     "debug": "4.3.1",
-    "esotope-hammerhead": "0.6.6",
+    "esotope-hammerhead": "https://github.com/PavelMor25/builds/raw/master/esotope-hammerhead-0.6.7.tgz",
```

Comment:
As I see your PR in `esotope-hammerhead` has already been merged, so you can publish a new version `esotope-hammerhead`, and set it here.